coap-gateway/coapconv/coapconv.go

```diff
@@ -34,7 +35,7 @@ func StatusToCoapCode(status commands.Status, operation Operation) codes.Code {
 		}
 	case commands.Status_CREATED:
 		return codes.Created
-	case commands.Status_ACCEPTED:
+	case commands.Status_NOT_MODIFIED:
```

Comment:
if `StatusToCoapCode` is called with  `commands.Status_ACCEPTED` it will now return `codes.BadRequest`. Is that correct behavior?

Reply:
fixed